web/src/main/webapp/WEB-INF/data/data/resources/map/config-viewer.xml

```diff
@@ -10,7 +10,7 @@
   <ows-context:ResourceList>
     <ows-context:Layer name="{type=mapquest}"
                        group="Background layers"
-                       hidden="false"
+                       hidden="true"
```

Comment:
From my point of view, I would say no, but some other probably like it as default ! The "Toner" version could also be nice to better view records extent. So I'm not sure we'll agree on the best default - we should maybe organize a poll ! :)